Mathlib/CategoryTheory/CofilteredSystem.lean

```diff
@@ -185,11 +185,13 @@ def toPreimages : J ⥤ Type v where
     rw [← mem_preimage, preimage_preimage, mem_preimage]
     convert h (g ≫ f); rw [F.map_comp]; rfl
   map_id j := by
-    simp (config := { unfoldPartialApp := true }) only [MapsTo.restrict, Subtype.map, F.map_id]
+    -- Adaptation note: nightly-2024-03-16: simp [Subtype.map]
```

Comment:
If we're going to write an adaptation note, I think we ought to mention that `unfoldPartialApp` was removed.

Reply:
If the new `simp only` doesn't have any downsides (e.g. apparently here?), I think just delete the adaptation note.

If there is some downside, the adaptation note should contain enough information for someone coming along afterwards to reconstruct the original proof and see if it works again.